lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -392,6 +392,9 @@ def load_config(self, commands, return_error=False, opts=None, replace=None):
         """Sends the ordered set of commands to the device
         """
         if replace:
+            device_info = self.get_device_info()
+            if '9K' not in device_info.get('network_os_platform'):
```

Comment:
This might result in exception if `network_os_platform` is not set. It can be written as `device_info.get('network_os_platform', ''`=)`